docs/logs/README.md

```diff
@@ -5,9 +5,9 @@
 The following tutorials have demonstrated the best practices for logging with
 OpenTelemetry .NET:
 
-* [Getting Started - Console Application](./getting-started-console/README.md)
 * [Getting Started - ASP.NET Core
   Application](./getting-started-aspnetcore/README.md)
+* [Getting Started - Console Application](./getting-started-console/README.md)
```

Comment:
Changed the order here to match https://github.com/reyang/opentelemetry-dotnet/tree/b5b1b4191e0402a9564adbe6454ef082a2a8f123?tab=readme-ov-file#getting-started.